src/site/content/en/blog/api-for-fast-beautiful-web-fonts/index.md

```diff
@@ -0,0 +1,162 @@
+
+- Self-host your own WOFF2 files.
+- Use the CSS API to retrieve the WOFF2.
+- Use the CSS API with the text= parameter set to “こんにちは.”
+
+{% Img src="image/Xyvh8LLq27V5yRjH5iS1dbf64pE2/B9FAwSaH2bJQ4SHCmRJk.png", alt="graph with comparison of different methods of downloading Noto Sans JP", width="800", height="324" %}
+
+In this example, you can see that by using the CSS API you are already saving 97.5% over self-hosting the WOFF2 font, thanks to the API’s built-in support for separating large fonts into unicode-range. By taking it a step further and specifying exactly the text you want to display, you can further reduce the size of the font to only 95.3% of the CSS API font – that’s 99.9% smaller than the self-hosted font.
+
+The Google Fonts CSS API will automatically deliver fonts in the smallest and most compatible format supported by your user’s browser. If your user is on a browser with WOFF2 support, the API will supply the fonts in WOFF2, but if they are using an older browser, the API will deliver the fonts in a format supported by that browser. To reduce the file size for each user, the API also removes data from the fonts when it’s not needed. For example, hinting data will be removed for users whose browsers do not need it.
+
+## Future-proofing your web fonts with the Google Fonts CSS API
+
+The Google Fonts team also contributes to new W3C standards that continue innovating web font technologies, such as <a href="https://www.w3.org/TR/WOFF2/">WOFF2</a>. One current project is <a href="https://www.w3.org/TR/PFE-evaluation/">Incremental Font Transfer</a>, which allows users to load very small parts of font files as they’re used on screen and stream in the rest on demand, surpassing the performance of unicode-range. When you use our web fonts API, your users get these underlying font transfer technology improvements when they become available in their browser.
+
+This is the beauty of a fonts API: your users have the benefit of each new technology improvement without any changes to your website. New web font format? No problem. New browser or operating system support? It's taken care of. So instead of being bogged down with the maintenance of your web fonts, you are free to focus on your users and your content. 
+
+## Variable fonts support built-in
+
+<a href="https://fonts.google.com/knowledge/glossary/variable_fonts">Variable fonts</a> are font files that can store a range of design variation amongst multiple <a href="https://fonts.google.com/knowledge/glossary/axis_in_type_design">axes</a>, and the <a href="https://developers.google.com/fonts/docs/css2">new version</a> of our CSS API includes support for them. Adding an additional axis of variation enables new flexibility with a font—but it can almost double the size of the font file.
```

Comment:
```suggestion
[Variable fonts](https://fonts.google.com/knowledge/glossary/variable_fonts) are font files that can store a range of design variation amongst multiple [axes](https://fonts.google.com/knowledge/glossary/axis_in_type_design), and the [new version](https://developers.google.com/fonts/docs/css2) of the Google Fonts CSS API includes support for them. Adding an additional axis of variation enables new flexibility with a font—but it can almost double the size of the font file.
```